ska_helpers/tests/test_git_helpers.py

```diff
@@ -0,0 +1,30 @@
+import pytest
+import tempfile
+from ska_helpers import git_helpers
+import git
+import os
+from pathlib import Path
+
+
+CHANDRA_MODELS = Path(os.environ["SKA"]) / "data" / "chandra_models"
```

Comment:
This should use `ska_helpers.paths.chandra_models_repo_path()`, which pays attention to environment vars that might update the path.

Reply:
I think it is likely that we don't actually want to pay attention to the environment vars for this test.

Reply:
Why not? In normal testing the suggestion above will give the same answer, but e.g. if you wanted to run tests on Mac where you have another account available, you could use `CHANDRA_MODELS_REPO_DIR` to point at that other version of `chandra_models` and then run the unit test.